pkg/idutil/netwalker/netwalker.go

```diff
@@ -53,6 +53,10 @@ func (w *NetworkWalker) Walk(ctx context.Context, req string) (int, error) {
 	}
 
 	idFilterF := func(n *netutil.NetworkConfig) bool {
+		if n.NerdctlID == nil {
+			// External network
+			return false
```

Comment:
NO LGTM, this should be fixed by https://github.com/containerd/nerdctl/pull/1618/files#diff-d14f39e1abb6264e177f986c3c0cc45b0250bdfe19c861c3bdaa4dfd4c50d6a0L58-R58